content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -72,6 +72,17 @@ To setup unified service tagging in a containerized environment:
 
 3. Configure your environment that corresponds to your container orchestration service based on either full configuration or partial configuration as detailed below.
 
+#### Automatic version tagging for containerized environments
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][16].
+
+Datadog sets the `version` tag for you if the following criteria are met:  
```

Comment:
it's a hard to parse. What do you think about adding a simple table with the order of priority and the value used?